CONTRIBUTING.md

```diff
@@ -4,33 +4,14 @@
 
 Use of virtual environments will allow for isolated installation of testing requirements:
 
-###Python 2
-
-```bash
-virtualenv -p python2.7 env27
-source env/bin/activate
-pip install -r test-requirements
-```
-
-###Python 3
-
 ```bash
-python3 -m venv env
+python -m venv env
```

Comment:
I'd keep python3 here, as it should always work, where python may still point to python2 in some environments.

https://www.python.org/dev/peps/pep-0394/#for-python-runtime-distributors